### DISCUSSION TITLE
How to use the core without an adapter

### DISCUSSION THREAD

**Discussion by alessandro308** [TanStack/query#5302] [Q&A]
I mean, if I want to develop my adapter for a custom framework, is there a documentation to use the core-only?

**Comment by TkDodo** [comment]
https://tkdodo.eu/blog/inside-react-query

**Reply by alessandro308** [reply]
I cannot use this post a docs to start to implement a custom adapter 😓 It's too high level view

**Reply by TkDodo** [reply]
what would you need then? you can also take a look at the 4 adapters that already exist and take it from there. It's literally: create an observer, subscribe to it, make sure your component re-renders when an event comes in ...
